joschahenningsen/TUM-Live
# Updated docs

### Motivation and Context
See #1370.

### Description
- Added new docs in `docs_v2` directory. See [github.com/facebook/docusaurus](https://github.com/facebook/docusaurus) for more details.
- Added new GitHub Workflow `deploy-docs.yaml` that automatically deploys dev branch docs and changed https://github.com/TUM-Dev/gocast/settings/pages to use `GitHub actions` as source.


### Steps for Testing
Prerequisites:
- Node >= 18 installed

Start the development server:
1. `cd docs_v2`
2. Run `npm i && npm run start`
3. Open localhost:3000/gocast and you should see the docs

### Screenshots
See https://docs.tum.live.
